variables.tf

```diff
@@ -175,6 +175,12 @@ variable "cluster_security_group_tags" {
   default     = {}
 }
 
+variable "include_cluster_security_group" {
```

Comment:
note: open to suggestions on naming here. was trying to come up with something that wasn't overly verbose